filechooser-portal/FileChooserDialog.vala

```diff
@@ -79,47 +79,53 @@ public class Files.FileChooserDialog : Hdy.Window, Xdp.Request {
     }
 
     construct {
-        previous_paths = new Queue<string> ();
-        next_paths = new Queue<string> ();
-        Hdy.init ();
+        previous_files = new Queue<GLib.File> ();
+        next_paths = new Queue<GLib.File> ();
 
         location_bar = new View.Chrome.BasicLocationBar ();
+        var title_label = new Gtk.Label (title);
+        var title_widget = new Gtk.Box (Gtk.Orientation.VERTICAL, 6);
+        title_widget.prepend (title_label);
+        title_widget.append (location_bar);
 
-        var previous_button = new Gtk.Button.from_icon_name ("go-previous-symbolic", Gtk.IconSize.LARGE_TOOLBAR) {
```

Comment:
When removing an icon size of `LARGE_TOOLBAR` we need to either add `Granite.STYLE_CLASS_LARGE_ICONS` or `pixel_size = 24`

Reply:
Done